### PR TITLE
Update manifest

### PR DESCRIPTION
**Pull request by rnmitchell**
Welp, I updated `setup.py` but forgot to update the manifest (ugh!).

### CODE REVIEW

**Comment by rnmitchell** [comment]
I'm going to address a few other issues in this PR as well.

**Comment by rnmitchell** [comment]
I've changed the code to use openpyxl to read in Excel files (to address #39). The code works but I get a Warning message when running ```make test```:
```
lusSTR/tests/test_format.py::test_format
lusSTR/tests/test_format.py::test_format_stdout
lusSTR/tests/test_format.py::test_format_sexloci_uas
  /Users/rneggebeeen/opt/anaconda3/envs/lusstr/lib/python3.7/site-packages/openpyxl/worksheet/_reader.py:308: UserWarning: Unknown extension is not supported and will be removed
    warn(msg)
```
I believe it's being caused by something within the file. It doesn't affect the output and the Warning isn't outputted when running the lusstr ```format``` command on the command line, only when running ```make test```.

**Comment by rnmitchell** [comment]
Should clarify that statement more... running the ```format``` command on the CL using the Sample Details Report used in the unit tests *does* produce the Warning message, but running ```format``` with a different Sample Details Report does not produce the Warning message.

**Comment by rnmitchell** [comment]
Not exactly sure what was up with that file, so I removed the old UAS Sample Details Report used in testing and replaced it with the UAS Sample Details Report file used in the SNP unit tests. All tests are passing now without any Warning messages.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
          ]
     },
     include_package_data=True,
-    install_requires=['pandas>=1.0', 'xlrd==1.2.0'],
+    install_requires=['pandas>=1.0', 'openpyxl>=3.0.6'],
```
Comment (standage) [inline setup.py:28]: Perhaps worth a comment in the README about what Excel file types are and aren't supported?

**Review by standage — APPROVED** [review]
Sorry this fell through the cracks. Looks good to me!

File: setup.py (hunk 1 above)
Comment (rnmitchell) [reply setup.py:28]: I can do that, but really all Excel files are coming from the UAS, so in theory, should be the same.

Comment (standage) [reply setup.py:28]: Ok, well probably not a concern then.

**Pull request merged**